go-update/src/test/java/org/reactome/release/goupdate/JavaVsPerlComparison.java

```diff
@@ -21,58 +26,98 @@
 {
 	public static void main(String[] args) throws Exception
 	{
-		MySQLAdaptor perlUpdatedDB = new MySQLAdaptor("localhost", "gk_central_2019_04_18", "root", "root", 3306);//new MySQLAdaptor("localhost", "gk_central_after_uniprot_update.dump.sql", "root", "root", 3308);
-		MySQLAdaptor javaUpdatedDB = new MySQLAdaptor("localhost", "gk_central_2019_04_18", "root", "root", 3307);//new MySQLAdaptor("localhost", "gk_central_R66_before_chebi_update.sql", "root", "root", 3307);
+		MySQLAdaptor perlUpdatedDB = new MySQLAdaptor("localhost", "perl_gk_central", "root", "root", 3306);
+		MySQLAdaptor javaUpdatedDB = new MySQLAdaptor("localhost", "java_gk_central", "root", "root", 3306);
```

Comment:
The formatting for created MySQLAdaptor objects differs from how ChebiUpdate does it: https://github.com/reactome/data-release-pipeline/pull/121/files/600bdc5e31fbe03661507750fb03f8b195feabe1..cf7396e26c9ffe9bf9acc8024dbddc376adc01d9#diff-3adf77f599ea429e20f4476c381fdaf0R21-R26

Reply:
Changed ChEBIUpdate to match: https://github.com/reactome/data-release-pipeline/pull/121/commits/19a51ca5c32d5b1f22091bcfdfdf0bcf58690153